src/test/datascience/notebook/introStartPage.vscode.test.ts

```diff
@@ -1,193 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I removed the tests, no point if this is going away and soon native notebook will be the default.